data/hai.txt

```diff
@@ -103,6 +143,47 @@ fleet "Large Hai"
 		"Shield Beetle (Pulse)"
 		"Shield Beetle (Tracker)"
 		"Lightning Bug" 2
+	variant
+		"Shield Beetle (Tracker)
+		"Centipede" 3
+	variant 2
+		"Water Bug" 3
+		"Centipede" 2
+	variant
+		"Centipede" 5
+	variant 3
+		"Centipede" 2
+		"Lightning Bug" 3
+		"Water Bug"
+	variant 2
+		"Aphid" 2
+		"Lightning Bug"
+		"Centipede"
+	variant 2
+		"Centipede"
+		"Shield Beetle (Pulse)" 2
+		"Shield Beetle (Tracker)"
+	variant
+		"Centipede" 2
+		"Lightning Bug" 2
+		"Aphid" 3
+	variant 
+		"Geocoris"
+		"Shield Beetle (Pulse)" 2
+		"Shield Beetle (Tracker)"
+	variant 
+		"Geocoris"
+		"Shield Beetle (Pulse)" 2
+		"Grasshopper" 2
+	variant 
+		"Geocoris"
+		"Shield Beetle (Pulse)" 
+		"Shield Beetle (Tracker)"
+		"Grasshopper" 2
+	variant 
```

Comment:
This is a REALLY big variant: 5 large ships and 3 small ones. The current largest variant among the large Hai fleets contains 2 Shield Beetles and 2 Lightning Bugs.

There are a few other unusually large variants above this one as well; the 3 above this variant are unusually large and so is the closest `variant 2` above this one.